uniffi/src/bindings/swift/templates/ObjectTemplate.swift

```diff
@@ -2,24 +2,26 @@ public class {{ obj.name() }} {
     private let handle: UInt64
 
     {%- for cons in obj.constructors() %}
-    public init({% call swift::arg_list_decl(cons.arguments()) -%}) {
+    public init({% call swift::arg_list_decl(cons) -%}) throws {
         self.handle = {% call swift::to_rs_call(cons) %}
     }
     {%- endfor %}
 
     // XXX TODO: destructors or equivalent.
 
+
+    // TODO: Maybe merge the two templates (i.e the one with a return type and the one without)
```

Comment:
Mmm, I don't mind that they're spelled out more explicitly, but we can go the other way and merge these, too! At first, I thought duplicating would be more readable than inline `match`es, but it might also mean we accidentally make a copy-pasta error between the two branches.